app/templates/events/view/tickets/order-form.hbs

```diff
@@ -1,66 +1,83 @@
-            </td>
+<form class="ui form {{if isLoading 'loading'}}" autocomplete="off">
+  <h2 class="ui header">
+    <div class="content">
+      {{t 'Order Form'}}
+    </div>
+  </h2>
+  <div class="spacer-10"></div>
+  <h3 class="ui dividing header">
+    <i class="checkmark box icon"></i>
+    <div class="content">
+      {{t 'Information to Collect'}}
+    </div>
+  </h3>
+  <div class="ui two column stackable grid">
+    <div class="column">
+      <table class="ui selectable celled table">
+        <thead>
+          <tr>
+            {{#if device.isMobile}}
+              <th class="center aligned">
+                {{t 'Options'}}
+              </th>
+            {{else}}
+              <th class="right aligned">
+                {{t 'Option'}}
+              </th>
+              <th class="center aligned">
+                {{t 'Include'}}
+              </th>
+              <th class="center aligned">
+                {{t 'Require'}}
+              </th>
+            {{/if}}
           </tr>
-        {{/each}}
-      </tbody>
-    </table>
-
-    <div class="ui buttons {{if device.isMobile 'fluid' 'right floated large'}}">
-      <button class="blue ui right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'save' model}}>
-        {{t 'Save'}}
-        <i class="save icon"></i>
-      </button>
+        </thead>
+        <tbody>
+          {{#each model.customForms as |field|}}
+            <tr class="{{if field.isIncluded 'positive'}}">
+              <td class="{{if device.isMobile 'center' 'right'}} aligned">
+                <label class="{{if field.isFixed 'required'}}">
+                  {{field.name}}
+                </label>
+              </td>
+              <td class="center aligned">
+                {{ui-checkbox class='slider'
+                              checked=field.isIncluded
+                              disabled=field.isFixed
+                              onChange=(action (mut field.isIncluded))
+                              label=(if device.isMobile (t 'Include'))}}
+              </td>
+              <td class="center aligned">
+                {{ui-checkbox class='slider'
+                              checked=field.isRequired
+                              disabled=field.isFixed
+                              onChange=(action (mut field.isRequired))
+                              label=(if device.isMobile (t 'Require'))}}
+              </td>
+            </tr>
+          {{/each}}
+        </tbody>
+      </table>
     </div>
   </div>
-</div>
\ No newline at end of file
+  <h3 class="ui dividing header">
+    <i class="options box icon"></i>
+    <div class="content">
+      {{t 'Registration Options'}}
+    </div>
+  </h3>
+  <div class="field">
+    <label>{{t 'REGISTRATION TIME LIMIT'}}</label>
+    <div class="{{unless device.isMobile 'two wide'}} field">
+      {{input type='number' name='orderExpiryTime' value=model.event.orderExpiryTime}}
```

Comment:
Should we have a limit on max value here?

Reply:
Yeah alright, I'll add it in the next PR. I also have to limit the input to only allow positive numbers. Need to use JS for that. Will also take care of this.